dolphin/db/sqlalchemy/models.py

```diff
@@ -121,3 +121,20 @@ class Disk(BASE, DolphinBase):
     model = Column(String(255))
     media_type = Column(String(255))
     capacity = Column(Integer)
+
+
+class AlertSource(BASE, DolphinBase):
+    """Represents an alert source configuration."""
+    __tablename__ = 'alert_source'
+    storage_id = Column(String(36), primary_key=True)
+    host = Column(String(255))
+    version = Column(String(255))
+    community_string = Column(String(255))
+    username = Column(String(255))
+    password = Column(String(255))
+    auth_enable = Column(Boolean)
+    auth_protocol = Column(String(255))
+    encryption_enable = Column(Boolean)
```

Comment:
rename it to encryption_enabled

Reply:
How about enable_encryption? I just want to keep the filed name be same as the one when user creating the alert source.

Reply:
Done